Makefile

```diff
@@ -2046,7 +2046,7 @@ SHA256_CMD = sha256sum
 
 ZLIB_VER ?= 1.2.11
 ZLIB_SHA256 ?= c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1
-ZLIB_DOWNLOAD_BASE ?= http://zlib.net
+ZLIB_DOWNLOAD_BASE ?= https://zlib.net/fossils
```

Comment:
I can see a reason for changing to "/fossils" but is there a reason why we change from http to https, considering the below "BZIP2_DOWNLOAD_BASE" still uses http?

Reply:
https is fine with me